packages/strapi-redis/lib/index.js

```diff
@@ -39,17 +40,27 @@ module.exports = function(strapi) {
         connector: 'strapi-redis'
       });
 
+      if(_.size(connections) === 0) {
+        cb();
+      }
+
       const done = _.after(_.size(connections), () => {
         cb();
       });
 
       // For each connection in the config register a new Knex connection.
       _.forEach(connections, (connection, name) => {
         // Apply defaults
-        _.defaults(connection.settings, strapi.hooks.redis.defaults);
+        _.defaults(connection.settings, strapi.hook.redis.defaults);
```

Comment:
`strapi.config.hook.settings.redis` should be better to keep consistency between hooks.